### PR TITLE
adjust ARIADNE template

### PR DESCRIPTION
**Pull request by fbenke-pik**
Adjust ARIADNE mapping with newly introduced variables in https://github.com/pik-piam/remind2/pull/435

This affects the Ariadne Variables
```
Capacity|Electricity|Coal
Capacity|Electricity|Coal|w/o CCS
Capacity|Electricity|Other
Secondary Energy|Electricity|Coal
Secondary Energy|Electricity|Other
Secondary Energy|Electricity|Coal|w/o CCS
```

### CODE REVIEW

**Comment by fbenke-pik** [comment]
Open Questions:

Do we create inconsistencies with the mappings concerning the REMIND variables
```
Cap|Electricity|Coal|w/ CC
Cap|Electricity|Coal|w/o CC
Cumulative Cap|Electricity|Coal
New Cap|Electricity|Coal
New Cap|Electricity|Coal|w/ CC
New Cap|Electricity|Coal|w/o CC
SE|Electricity|Coal|+|w/ CC
SE|Electricity|Coal|+|w/o CC
```
 
or the ARIADNE variable `Secondary Energy|Electricity|Fossil`?

@robertpietzcker

**Comment by robertpietzcker** [comment]
yes. 
change one thing, everything falls apart ...

so:
1. 
Secondary Energy|Electricity|Fossil|w/o CCS and Secondary Energy|Electricity|Fossil
need an additional summand (so - if I understand the logic correctly - an additional line), namely
`SE|Electricity|Other Fossil|Other Fossil Adjusted`  (should be added to both values, as it is w/o CC)


2.  this line:
`2;energy (electricity);Capacity|Electricity|Other;GW;Total installed (available) net capacity of all others types of operating power plants;;;;;0;;`

needs to change so that it reports the new "other fossil" capacity value (`Cap|Electricity|Other Fossil|Other Fossil Adjusted`) instead of 0 

3. Cap|Electricity|Coal|w/ CC is unchanged

4. Cap|Electricity|Coal|w/o CC needs a new variable, "Cap|Electricity|Coal|w/o CC|Other Fossil Adjusted (GW)"

5. Cumulative Cap|Electricity|Coal and NewCap are irrelevant for ARIADNE, as no non-CCS coal is built after 2015

File: inst/templates/mapping_template_ARIADNE.csv [hunk 1]
```diff
@@ -1184,11 +1184,13 @@ Tier;Category;Variable;Unit;Definition;piam_variable;piam_unit;piam_factor;Comme
 1;energy (electricity);Secondary Energy|Electricity|Fossil;PJ/yr;net electricity production from coal, gas, conventional and unconventional oil;SE|Electricity|+|Coal;EJ/yr;1000;;0;;
 1;energy (electricity);Secondary Energy|Electricity|Fossil;PJ/yr;net electricity production from coal, gas, conventional and unconventional oil;SE|Electricity|+|Gas;EJ/yr;1000;;0;;
 1;energy (electricity);Secondary Energy|Electricity|Fossil;PJ/yr;net electricity production from coal, gas, conventional and unconventional oil;SE|Electricity|+|Oil;EJ/yr;1000;;0;;
+1;energy (electricity);Secondary Energy|Electricity|Fossil;PJ/yr;net electricity production from coal, gas, conventional and unconventional oil;SE|Electricity|Other Fossil|Other Fossil Adjusted;EJ/yr;1000;;0;;
```
Comment (robertpietzcker) [inline inst/templates/mapping_template_ARIADNE.csv:1187]: as three lines above, SE|Electricity|+|Coal is used and not SE|Electricity|Coal|Other Fossil Adjusted, this line here (1187) can actually be deleted again. 
sorry

File: inst/templates/mapping_template_ARIADNE.csv [hunk 2]
```diff
@@ -1184,11 +1184,13 @@ Tier;Category;Variable;Unit;Definition;piam_variable;piam_unit;piam_factor;Comme
...
 2;energy (electricity);Secondary Energy|Electricity|Fossil|w/ CCS;PJ/yr;net electricity production from coal, gas, conventional and unconventional oil used in combination with CCS;SE|Electricity|Gas|+|w/ CC;EJ/yr;1000;;0;;
 2;energy (electricity);Secondary Energy|Electricity|Fossil|w/o CCS;PJ/yr;net electricity production from coal, gas, conventional and unconventional oil without CCS;SE|Electricity|Coal|+|w/o CC;EJ/yr;1000;;0;;
 2;energy (electricity);Secondary Energy|Electricity|Fossil|w/o CCS;PJ/yr;net electricity production from coal, gas, conventional and unconventional oil without CCS;SE|Electricity|Oil|w/o CC;EJ/yr;1000;;0;;
 2;energy (electricity);Secondary Energy|Electricity|Fossil|w/o CCS;PJ/yr;net electricity production from coal, gas, conventional and unconventional oil without CCS;SE|Electricity|Gas|+|w/o CC;EJ/yr;1000;;0;;
+2;energy (electricity);Secondary Energy|Electricity|Fossil|w/o CCS;PJ/yr;net electricity production from coal, gas, conventional and unconventional oil without CCS;SE|Electricity|Other Fossil|Other Fossil Adjusted;EJ/yr;1000;;0;;
```
Comment (robertpietzcker) [inline inst/templates/mapping_template_ARIADNE.csv:1193]: as above. 
I thought wrongly before that the SE|Elec|Coal value changed, but as it is unchanged and only the SE|Elec|CoalFossil adjusted is smaller, this line (1193) can again be deleted

**Pull request merged**